README.md

```diff
@@ -14,6 +14,35 @@ bundle exec rake demo:server  # runs both Rails and Solr
 bundle exec rake demo:seed    # to load data from data/ead folder
 ```
 
+## Indexing sample data
+Sample files are included in the data directory. To index sample data:
```

Comment:
so to index all the data, you'd run all 9 commands?

Reply:
yes